### PR TITLE
Added New Resources and Removed one broken link

### PR DESCRIPTION
**Pull request by aaryan2134**
I have added 2 resources for Learning Resources category and 1 student discount category. 
Please see if the changes are okay.

Also If you could please consider this PR for Hacktober Fest that would be great!
Thanks!

Let me know If there are any changes required.

### CODE REVIEW

**Comment by aaryan2134** [comment]
@closebracket Hi, Could you please have a look at this PR? Let me know if there is any issue

**Comment by closebracket** [comment]
> @closebracket Hi, Could you please have a look at this PR? Let me know if there is any issue

Hello @aaryan2134,

Thank you for submitting to Freeside's Student Resource List under Hacktoberfest.

Unfortunately, your initial commit has a number of mark down formatting issues. For instance, you have formatted the contents header as a link and changed one of the links to include dashes (---) where none were required.

Similarly, I am sceptical of the quality of resources submitted. In particular, Scihub, which, whilst I personally agree that such content _should_ be free, adding a resource to the list that is legally/ethically  ambiguous could get Freeside into trouble with the University of Hull so it would not be considered a good quality resource.

If you could contribute other resources and correct your formatting issues, I would be happy to re-review.

Naturally, if you are unhappy with my review. You are more than welcome to request one from Freeside Admins, such as @sbrl @fisherthewol and @admgecko.

**Comment by aaryan2134** [comment]
@closebracket Oh, sorry that was a slight error while making the changes. I will correct that right way. 
I understand your point. I will remove scihub.

**Comment by aaryan2134** [comment]
@closebracket Could you please have a look again?
I have made the required changes.
Thanks!

File: README.md [hunk 1]
```diff
@@ -105,6 +104,7 @@ If you require any support, either whilst contributing or with linux, programmin
 8. [Dockstation](https://dockstation.io/) - Docker GUI for Windows, Mac and Linux
 9. [Tush](https://github.com/darius/tush/) - a literate testing shell
 10. [Haven Self Hosted Blog](https://github.com/havenweb/haven) - A self hosted private blog built with ruby on rails.
+11. [Fritzing](https://fritzing.org/) - A software for desinging circuits, PCBs and testing them virtually.
```
Comment (closebracket) [inline README.md:107]: Fritzing is good. This resource will be accepted.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,23 @@
 ![Freeeside Logo](https://camo.githubusercontent.com/8824210474b9586405f0b11e210ea266599c9f83/68747470733a2f2f63646e2e7261776769742e636f6d2f467265657369646548756c6c2f4c6f676f732f6d61737465722f66726565736964655f7371756172655f706174682e737667 "Freeside Logo")
 
 ## Contents
```
Comment (closebracket) [inline README.md]: Good.

File: README.md [hunk 3]
```diff
@@ -146,7 +146,6 @@ If you require any support, either whilst contributing or with linux, programmin
 26. [How To Build An App: Everything You Didn't Know You Needed To Know](https://youtube.com/playlist?list=PL96C35uN7xGJu6skU4TBYrIWxggkZBrF5)
 27. [Jeff Geerling](https://www.youtube.com/c/JeffGeerling) - YouTube channel with lots of Raspberry Pi projects and tests, along with resources on Kubernetes, Ansible and more.
 28. [Data Structures and Algorithms Visualiser](https://visualgo.net/en) - Provides visulation for different data structures and algorithms like sorting, linked lists, hash table etc. Also allows some customizations.
-29. [Scihub](https://sci-hub.se/) - To remove all barriers in the way of science. Gives access to research papers to students and researchers to study and learn.
```
Comment (closebracket) [inline README.md]: Thank you for removing this resource.

**Review by closebracket — APPROVED** [review]
Thank you for making the required changes. I will approve this PR for Hacktoberfest because upon checking the rules, I note that I did not specify a minimum limit for resources except for if a new category / topic was started. For that reason, your contribution should be accepted because you have abided by the rules - but I will update the rules after this.

**Pull request merged**